packages/compass-collection/src/components/collection-header/collection-header.tsx

```diff
@@ -124,42 +124,59 @@ type CollectionHeaderProps = {
 
 const ATLAS_SEARCH_LP_LINK = 'https://www.mongodb.com/cloud/atlas/lp/search-1';
 
+const INSIGHTS = {
+  'atlas-text-regex-usage-in-view': {
+    id: 'atlas-text-regex-usage-in-view',
+    title: 'Alternate text search options available',
+    description:
+      "In many cases, Atlas Search is MongoDB's most efficient full text search option. Convert your view’s query to $search for a wider range of functionality.",
+    learnMoreLink:
+      'https://www.mongodb.com/docs/atlas/atlas-search/best-practices/',
+    primaryActionButtonLink: ATLAS_SEARCH_LP_LINK,
+    primaryActionButtonLabel: 'Try Atlas Search',
+  },
+  'non-atlas-text-regex-usage-in-view': {
+    id: 'non-atlas-text-regex-usage-in-view',
+    title: 'Alternate text search options available',
+    description:
+      "In many cases, Atlas Search is MongoDB's most efficient full text search option. Connect with Atlas to explore the power of Atlas Search.",
+    learnMoreLink:
+      'https://www.mongodb.com/docs/atlas/atlas-search/best-practices/',
+    primaryActionButtonLink: ATLAS_SEARCH_LP_LINK,
+    primaryActionButtonLabel: 'Try Atlas Search',
+  },
+  'lookup-in-view': {
+    id: 'lookup-in-view',
+    title: '$lookup usage',
+    description:
+      '$lookup operations can be resource intensive because they perform operations on two collections instead of one. In certain situations, consider embedding documents or arrays to enhance read performance.',
```

Comment:
(same as above, should they be deduplicated somehow?)

Reply:
Hmmm, I'd still need to keep the actions separate because they are different in different plugins, but probably makes sense to move all the static parts to one place, I'll do this after we merge all the currently open insights to avoid conflicts